SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -488,10 +550,10 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 		}
 
 		FSpatialGDKSpanId SpanId;
-		if (EventTracer != nullptr)
+		if (EventTracer)
 		{
-			SpanId = EventTracer->TraceEvent(FSpatialTraceEventBuilder::CreateSendPropertyUpdates(Object, EntityId, Update.component_id),
-											 CauseSpanId.GetConstId(), 1);
+			SpanId = EventTracer->TraceEvent(FSpatialTraceEventBuilder::CreateSendPropertyUpdate(Object, EntityId, Update.component_id),
```

Comment:
Hmm maybe we don't need this anymore? We have an event per property change.

Reply:
I think the problem is not being able to send in an array to `SendComponentUpdate`, we don't have a way to combine causes without an event anymore?

Reply:
We could just add that though if it makes sense

Reply:
There is an additional point here that we have the component_id whereas before create traces without that